test_collections/matter/setup.sh

```diff
@@ -33,6 +33,9 @@ for package in ${packagelist[@]}; do
 done
 IFS=$SAVEIFS 
 
+print_script_step "Pulling chip-cert-bins docker image"
+$MATTER_PROGRAM_DIR/scripts/update-pull-sdk-docker-image.sh
+
```

Comment:
As discussed offline: reminder to move this script call to `update-sample-apps.sh`